### PR TITLE
Update master

### PR DESCRIPTION
**Pull request by frederikja163**
Updates the master branch, and makes some of the most recent changes go live on the website

### CODE REVIEW

File: api/index.html [hunk 1]
```diff
@@ -54,65 +54,58 @@
...
-</html>
+</article>
+          </div>
+          
+          <div class="hidden-sm col-md-2" role="complementary">
+            <div class="sideaffix">
+              <div class="contribution">
+                <ul class="nav">
+                  <li>
+                    <a href="https://github.com/frederikja163/opentk.net/blob/docfx/api/index.md/#L1" class="contribution-link">Improve this Doc</a>
```
Comment (jvbsl) [inline api/index.html]: I think you should work directly in the main repository, as it uses your repository path, instead of the original one

Comment (varon) [reply api/index.html]: Thanks for figuring this all out @frederikja163 !

Great spot @jvbsl.

Comment (frederikja163) [reply api/index.html]: is this the only spot? because then it would be a simple manual fix, otherwise i will take a look at it

Comment (jvbsl) [reply api/index.html]: no there should be some more, but after spotting that I only searched for "freder"^^

Comment (frederikja163) [reply api/index.html]: Outdated

**Comment by jvbsl** [comment]
I think it should be fine now

**Pull request merged**